app/controllers/api/comments_controller.rb

```diff
@@ -13,10 +13,12 @@ def create
 
     author = post.author
 
-    comment = post.comment.new(text:, author:)
+    comment = post.comments.new(text:, author:)
 
-    return unless comment.save
-
-    render json: comment
+    if comment.save
+      render json: comment, status: :created
+    else
+      render json: { errors: comment.errors.full_messages }, status: :unprocessable_entity
+    end
```

Comment:
@AndreaM2429 Great job! It is a good practice to implement validations when generating a response from insertion, deletion, or update, no matter what type of backend it is (in this case an API) and you have done it very well.